src/dns/coredns.rs

```diff
@@ -256,7 +265,14 @@ impl CoreDns {
                                     }
                                 }
                             }
-                            let record_name: Name = Name::from_str_relaxed(name.as_str()).unwrap();
+                            let record_name: Name = match Name::from_str_relaxed(name.as_str()) {
+                                Ok(name) => name,
+                                Err(e) => {
+                                    // log and continue server
+                                    debug!("Error while parsing record name: {:?}", e);
```

Comment:
should this be an error vs debug?

Reply:
We don't have any error message but we received `None`. But I agree this should be on `error!`